README.md

```diff
@@ -115,31 +113,20 @@ This repository is also configured with "EditorConfig" to make sure everyone's f
 
 Below are some useful commands that can be executed from the root of this repository in your terminal.
 
-- `./gradlew clean`: Removes `build` directory so that you have a fresh build result.
+- `./gradlew clean`: Removes `build` and `dist` directory so that you have a fresh build result.
```

Comment:
Why would there be both `build` and `dist`? Can we change the setup so that `dist` is not needed?

Reply:
The `build` directory comes out of the box setup of Gradle because of the `base` plugin.

Reply:
Replaced `dist` with `build`.